lib/cassia/access_controller.rb

```diff
@@ -74,6 +74,20 @@ def combined_sse
 
     def ac_url
       Cassia.configuration.ac_url
+    def discover_all_services(router: , device_mac: )
```

Comment:
I think you didn't save and commit these changes to fix the access controller. 

We need to add these functions above the private label and add and `end` to `ac_url`